academic-requirements/src/App.js

```diff
@@ -30,6 +30,8 @@ function App() {
   const [major, setMajor] = useState("");
   const [concentration, setConcentration] = useState("");
 
+  const [coursesTaken, setCoursesTaken] = useState([]);
+
```

Comment:
What is the difference between this and the previouslyTakenCourses that I see in the main branch? It seems like that got replaced here, any ideas what happened there? (Mostly just noting so we make sure we don't end up with two arrays when we merge this)

Reply:
Yeah that makes sense, I'll get that changed.

Reply:
CoursesTaken was the name of the variable that stored the taken courses selected from the input page in the InputPage.tsx file, https://github.com/CS-458/Academic_Requirements/blob/main/academic-requirements/src/components/InputPage.tsx#L46, so I used coursesTaken in App.JS to keep the naming convention consistent. So, I don't think there really is much different looking at it and could possibly change the variable names if needed.

Reply:
The name change is totally fine (consistency is good), I just want to make sure that everything using it is updated when we merge, but it looks like the old name wasn't being used much so it shouldn't be a huge problem.